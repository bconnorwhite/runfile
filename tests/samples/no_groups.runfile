# Commands without groups - all at root level

# Simple command
hello:
  echo "Hello, World!"

# Command with args
greet
  name?: # Optional name parameter
  echo "Hello, ${name:-World}!"

# Command with flags
build
  --debug     # Enable debug mode
  --release:  # Build in release mode
  echo "Building with debug: $debug"
  echo "Release mode: $release"
