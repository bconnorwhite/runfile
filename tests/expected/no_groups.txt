hello         # Simple command
greet         # Command with args
  name?         # Optional name parameter
build         # Command with flags
  --debug       # Enable debug mode
  --release     # Build in release mode

